src/ui/components/navigation/navigationcomponent.js

```diff
@@ -237,6 +237,19 @@ export default class NavigationComponent extends Component {
       this.refitNav();
       DOM.on(DOM.query(this._container, '.yxt-Nav-more'), 'click', this.toggleMoreDropdown.bind(this));
     }
+    const navLinks = DOM.queryAll(this._container, '[data-originalurl]');
+    navLinks.forEach(link => {
+      const originalUrl = link.dataset.originalurl;
+      if (originalUrl) {
+        DOM.on(link, 'click', e => {
+          if (e.metaKey) {
```

Comment:
I thought `metaKey` would only watch for `Cmd` on Mac. Does this also catch middle click, `Ctrl-click` on Windows, and opening a new tab through the context menu?

Reply:
Ah yeah I tested and on windows it needs to be ctrlKey. For the others, middle click doesn't trigger onclick handlers and the new tab through context menu behavior shouldn't be changed

Reply:
if we don't explicitly check here for the new tab being opened through the context menu, wouldn't the new tab open the iFrame directly instead of having it within the parent domain?

Reply:
The new tab will use the parent url from setParentUrl instead of the original url